supfile.go

```diff
@@ -26,6 +26,7 @@ type Network struct {
 	Env       EnvList  `yaml:"env"`
 	Inventory string   `yaml:"inventory"`
 	Hosts     []string `yaml:"hosts"`
+	SSHKey    string   `yaml:"ssh-key"`
```

Comment:
We're trying to avoid dashes and underscores in the Supfile API. Can we think of one word here?

identity ... or sshkey ... any other suggestions?